content/js1/sprints/3/prep/index.md

```diff
@@ -8,4 +8,634 @@ backlog= 'Module-JS1'
 backlog_filter= 'Week 3'
 +++
 
+## Pre-requisites
 
+## Learning objectives
+
+- Write an assertion for a function that implements a solution to a given problem
+- Given an assertion and a function declaration, identify and explain whether the assertion will pass or fail
+- Explain when to use the `===` operator in a program
+- Construct a flow diagram that describes how to produce an output from a given input
+- Explain when it is necessary to use a conditional statement
+- Identify the syntactic features of an `if` statement
+- Describe the difference between `console.log` and `console.assert`
+- Evaluate an expression that uses a comparison operator
+- Describe how to convert a string into a number in JavaScript
+- Implement a piece of functionality specified in an assertion
+- Define _block scope_
+- Define _zero-indexing_
+- Given a short program, determine the return value of a call to `slice` before execution
+- Given a problem, suggest potential sub-goals required to reach a solution
+- Determine the output of a program that uses a conditional statement
+- Analyse a conditional statement to determine whether or not the statement will be executed
+- Predict and explain how conditional statements involving `else`, `else if` and `while` will be executed
+- Explain why _refactoring_ is necessary in the production of software
+- Propose a strategy for solving a problem
+- Write an assertion to check for some behaviour of a given function
+- Identify a pattern between a set of inputs and outputs produced by a given function
```

Comment:
I'd add some LOs around identifying classes of input data, and edge-case identification.